src/server_manager/electron_app/release_macos_action.sh

```diff
@@ -21,16 +21,48 @@ if [ "$CSC_LINK" == "unset" ]; then
   exit 1
 fi
 
+function usage () {
```

Comment:
nit: can we can we move this code to a separate script (i.e. `get_packaging_opts.sh`) and call it from all the scripts?